euth/reports/emails.py

```diff
@@ -0,0 +1,72 @@
+from django.contrib.auth import get_user_model
+from django.contrib.sites.models import Site
+from django.core import urlresolvers
+from django.db.models import ForeignKey
+from django.template import Context
+
+from euth.contrib import emails
+
+
+def send_email_to_moderators(request, report):
+
+    obj = report.content_object
+
+    if obj:
+        name = obj._meta.verbose_name.title()
+
+        try:
+            admin_url = urlresolvers.reverse("admin:%s_%s_change" %
```

Comment:
I would suggest to use `format()` instead of `%`. This would allow something like this:
```python
"admin:{m.app_label}_{m.model_name}_change".format(m=obj._meta)
```
IMO `format()` has a syntax that is easier to read and understand.